src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -46,13 +38,14 @@ const NavHeader = (props) => {
     onCancelToken,
     placeholderText,
     setIsFirstTimeOpen,
-    getSearchLayout,
-    opacityImage
+    getSearchLayout
   } = props;
   const navigation = useNavigation();
   const coverPath = topicDetail?.cover_path || null;
-  const insets = useSafeAreaInsets();
-  const {width: displayWidth} = useWindowDimensions();
+  const [domainHeight, setDomainHeight] = React.useState(0);
+  const [searchHeight, setSearchHeight] = React.useState(0);
+  const {top} = useSafeAreaInsets();
+  const topPosition = Platform.OS === 'ios' ? top : 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `opacityImage` and `useWindowDimensions` could potentially affect the layout and appearance of the component. Make sure to test thoroughly on different screen sizes and orientations to ensure the UI still behaves as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->